ApollosProject/apollos-prototype
# Adds ui config to apollos config package

## DESCRIPTION

### What does this PR do, or why is it needed?

Adds a different entry point for the `@apollosproject/config` package. This rn specific package variant will be used to share configuration (aka fragments) across all the different Apollos UI packages. It is also loads the data from RN Config into it's own configuration, so you don't have to use two different packages.

### What design trade-offs/decisions were made?

I would have loved to share the `config` code directly, instead of introducing a variant, but the different npm requirements made that tricky. 

### How do I test this PR?

Open the app, check that your app can still hit the server (confirms that the env variables are being loaded correctly)

### What automated tests did you write?

Tests over new module.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
